packages/appsync-emulator-serverless/serverCore.js

```diff
@@ -31,18 +31,28 @@ class SubscriptionServer {
     mqttServer.on('clientDisconnected', (...args) =>
       this.onClientDisconnect(...args),
     );
+
+    // XXX: Note we may need to listen to unsubscribe to end the async iterator.
```

Comment:
Agreed.

In the request for price tool I can foresee a customer potentially unsubscribing when the current transaction is completed.

Reply:
I am going to double check but I think this current implementation is fine ... We essentially end up with one connection per subscription . I will verify that works.